schema-engine/sql-introspection-tests/tests/postgres/constraints.rs

```diff
@@ -4,7 +4,464 @@ use indoc::indoc;
 use sql_introspection_tests::test_api::*;
 use test_macros::test_connector;
 
-#[test_connector(tags(Postgres), exclude(CockroachDb))]
+#[test_connector(tags(Postgres))]
+async fn aragon_test_postgres(api: &mut TestApi) -> TestResult {
+    let raw_sql = indoc! {r#"
+      CREATE TABLE eventsources (
```

Comment:
Do we really need all of this SQL to reveal the breakage? Could we have a minimal reproduction?

Reply:
Also, indentation is four spaces in Rust :)